keras_hub/src/utils/openvino_utils_test.py

```diff
@@ -0,0 +1,386 @@
+import os
+import tempfile
+import unittest.mock
+
+import keras
+import numpy as np
+import pytest
+
+from keras_hub.src.tests.test_case import TestCase
+
+try:
+    import openvino as ov
```

Comment:
for tests, unlike for the library proper, we can assume we have all the dependencies in requirements-common.txt. OK to just do normal import for tests.